### ISSUE TITLE
maybe wrong IPA symbol for a German word "aneinander"

### ISSUE THREAD

**Issue by XuesongYang** [espeak-ng/espeak-ng#1637]
Previously reported to Phonemizer repo: https://github.com/bootphon/phonemizer/issues/142, but suggested to re-direct here.

There is `/1/` appearing in the phonetic transcription for the German language. Is it expected?
```
$ echo "aneinander" | phonemize -l de -b espeak --with-stress
ˌan1aɪnˈandɜ
```

**Comment by jaacoppi** [comment]
Expected. It is define in the dictionary list:
dictsource/de_list:625:aneinander	an1aIn'and3

Someone with fluency in German phonetics can comment if this is correct or not.

**Comment by hozosch** [comment]
Yes, it was me who added this to the list. Using 1 creates a small gap in the speech between "an" and "einander", which I thought to be necessary. But now that I think  about it, it's not really. Besides, I could have taken care of it differently. So, I'll open a pull request where I remove the entry in the list and correct the rule for "einander" at the end of words, because the stress in that rule is not correct, it should fall on "ander".

**Comment by hozosch** [comment]
Strange, I cannot find the rule I was looking for. And still, "auseinander" is not stressed the way I want.

**Comment by hozosch** [comment]
There's the rule:
        ein (and   _|%aIn
Which, as it's defined now, I would expect to make the stress fall on "ander", but it doesn't.

**Comment by XuesongYang** [comment]
thank you @jaacoppi @hozosch to dive deep. Is there any PR that I can check further? I don't know much about the German language, but `/1/` seems more like a lexical stress marker.

**Comment by hozosch** [comment]
I can see why you don't like the 1, it can be replaced with _|, which is more standard. But the gab there, in case of ESpeak, sounds somehow better to me. Many of the words I added back then were more done from a perspective of sound, rather than correct phoneme symbols, but replacing the 1 with _| would not change the sound in the least. I didn't know of that alternative back then. I'm just not sure if I like the idea of no gab tere. The other words where you fixed the o weren't added by me by the way. I don't know who did that.

**Comment by hozosch** [comment]
I know that both  variants (with and without pause) are valid, but in the case of ESpeak, the pause sounds somehow better to me. I use ESpeak without Mbrola or Piper or something like that. I agree though, in natural and fast spoken speech, it sounds better without the pause. Espeak normally speaks really clearly though, and that's also why I don't like how Hypothenusenabschnitt sounds, as it doesn't fit into the rest of the system. Maybe Hypothenuse should be added to the rules, or the list entry should be changed, to use @n instead of n- and _| instead of the other glottal stop symbol. I'm sorry for being off topic there, but I went on a tangent there.

**Comment by hozosch** [comment]
I meant the current pronunciation in master, and there not even the "hypo" part, I meant the "en" in hypothenusen. Someone tried to make ESpeak sound like normal people would say it, but that's not how ESpeak works. I'm just going to put how it is now, and how I would prefer it.
Hypothenuse was already correct in your pr.
Hypothenusen: current way: hypo'te'nu:zn-, better: hypo:te:n'u:z@n
For Hypothenusenabschnitt, see above, and the  pause between the two words should be done with_|, not ?.
BTW the $4 can be removed as the apostrophy symbols already determine the stress. Those flags, dollar  + syllable number, are only useful if the stress is the only issue, then you can correct the stress without writing the whole word out in phonemes.
